src/components/Btn/Btn.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import styles from './btn.module.scss';
+import IBtn from '@/model/components/Btn/Btn';
+
+const Btn: React.FC<IBtn> = ({ className, ...props }): JSX.Element => {
```

Comment:
Actually, it's bad practice to move component props in another file. You have to switch between them and a interface is needed only in component file. So I'd like to leave BtnProps here.
+ Use fullname for components. A new developer will try to find `button` in the project, not `btn, Btn, bttn, etc.`